ccl/importccl: TestProtectedTimestampsDuringImportInto failed

[(ccl/importccl).TestProtectedTimestampsDuringImportInto failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1934953&tab=buildLog) on [master@e6b47088aba9c9978501b966a0e88aeb273d9990](https://github.com/cockroachdb/cockroach/commits/e6b47088aba9c9978501b966a0e88aeb273d9990):

```
I200511 11:32:28.598225 420042 util/stop/stopper.go:539  quiescing
I200511 11:32:28.599525 420041 util/stop/stopper.go:539  quiescing
I200511 11:32:28.605463 420043 util/stop/stopper.go:539  quiescing
I200511 11:32:28.639836 409058 kv/kvclient/kvcoord/transport_race.go:108  transport race promotion: ran 75 iterations on up to 6623 requests
E200511 11:32:28.641415 410337 jobs/registry.go:507  error while adopting jobs: failed querying for jobs: adopt-job: node unavailable; try another peer
W200511 11:32:28.642341 409076 jobs/registry.go:1230  job 554193180936994817: canceling due to liveness failure
W200511 11:32:28.642777 409076 jobs/registry.go:1230  job 554193204360577025: canceling due to liveness failure
W200511 11:32:28.663033 420082 kv/txn.go:607  [n1] failure aborting transaction: node unavailable; try another peer; abort caused by: job 554193204360577025: can't query system.jobs: job-status: context canceled
I200511 11:32:28.663484 408862 kv/kvserver/queue.go:579  [n1,s1] rate limited in MaybeAdd (gc): node unavailable; try another peer
I200511 11:32:28.664084 408862 kv/kvserver/queue.go:579  [n1,s1] rate limited in MaybeAdd (merge): node unavailable; try another peer
W200511 11:32:28.669378 408813 kv/txn.go:607  [n1,replication-reporter] failure aborting transaction: node unavailable; try another peer; abort caused by: upsert-replication-stats: node unavailable; try another peer
E200511 11:32:28.670105 408813 kv/kvserver/reports/reporter.go:139  [n1,replication-reporter] failed to generate replication reports: failed to save range status report: upsert-replication-stats: node unavailable; try another peer
I200511 11:32:28.670584 408862 kv/kvserver/queue.go:579  [n1,s1] rate limited in MaybeAdd (split): node unavailable; try another peer
I200511 11:32:28.671405 408862 kv/kvserver/queue.go:579  [n1,s1] rate limited in MaybeAdd (replicate): node unavailable; try another peer
I200511 11:32:28.672971 408862 kv/kvserver/queue.go:579  [n1,s1] rate limited in MaybeAdd (replicaGC): node unavailable; try another peer
E200511 11:32:28.675538 420086 kv/kvclient/kvcoord/txn_interceptor_committer.go:449  making txn commit explicit failed for "unnamed" meta={id=ecc27a94 key=/Table/11/1/52/3/2020-05-11T11:37:13.821537Z/1/0 pri=100.00000000 epo=0 ts=1589196747.556793305,0 min=1589196747.556793305,0 seq=2} lock=true stat=STAGING rts=1589196747.556793305,0 wto=false max=1589196747.556793305,0 ifw=1: result is ambiguous (server shutdown)
W200511 11:32:28.677442 410611 kv/kvserver/raft_transport.go:637  [n2] while processing outgoing Raft queue to node 1: EOF:
W200511 11:32:28.678665 410539 kv/kvserver/raft_transport.go:637  [n3] while processing outgoing Raft queue to node 1: EOF:
W200511 11:32:28.679279 415847 kv/kvserver/raft_transport.go:637  [n3] while processing outgoing Raft queue to node 2: EOF:
I200511 11:32:28.679812 408862 kv/kvserver/queue.go:579  [n1,s1] rate limited in MaybeAdd (raftlog): node unavailable; try another peer
I200511 11:32:28.694517 408862 kv/kvserver/queue.go:579  [n1,s1] rate limited in MaybeAdd (raftsnapshot): node unavailable; try another peer
I200511 11:32:28.695295 408862 kv/kvserver/queue.go:579  [n1,s1] rate limited in MaybeAdd (consistencyChecker): node unavailable; try another peer
I200511 11:32:28.696054 408862 kv/kvserver/queue.go:579  [n1,s1] rate limited in MaybeAdd (timeSeriesMaintenance): node unavailable; try another peer
E200511 11:32:28.680594 410135 jobs/registry.go:507  error while adopting jobs: failed querying for jobs: adopt-job: node unavailable; try another peer
W200511 11:32:28.682677 410755 kv/kvserver/raft_transport.go:637  [n3] while processing outgoing Raft queue to node 1: EOF:
W200511 11:32:28.684086 410645 kv/kvserver/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 2: EOF:
W200511 11:32:28.699508 415725 kv/kvserver/raft_transport.go:637  [n2] while processing outgoing Raft queue to node 3: EOF:
W200511 11:32:28.691452 415721 kv/kvserver/raft_transport.go:637  [n2] while processing outgoing Raft queue to node 3: rpc error: code = Unavailable desc = transport is closing:
I200511 11:32:28.704034 420085 rpc/nodedialer/nodedialer.go:160  [intExec=job-status,n1,txn=0c913d67] unable to connect to n3: failed to connect to n3 at 127.0.0.1:46301: context canceled
W200511 11:32:28.705329 410730 kv/kvserver/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 3: rpc error: code = Canceled desc = grpc: the client connection is closing:
E200511 11:32:28.707172 409078 jobs/registry.go:507  error while adopting jobs: job 554193180936994817: can't query system.jobs: job-status: failed to send RPC: sending to all 3 replicas failed; last error: <nil> failed to connect to n3 at 127.0.0.1:46301: context canceled
I200511 11:32:28.708397 420070 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:43493: context canceled
W200511 11:32:28.711884 410580 kv/kvserver/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 2: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200511 11:32:28.712916 415740 kv/kvserver/raft_transport.go:637  [n3] while processing outgoing Raft queue to node 2: EOF:
W200511 11:32:28.713868 419867 kv/kvserver/intentresolver/intent_resolver.go:744  failed to gc transaction record: could not GC completed transaction anchored at /Table/28/1/2/0: node unavailable; try another peer
W200511 11:32:28.718570 409917 gossip/gossip.go:1525  [n2] no incoming or outgoing connections
W200511 11:32:28.719507 419918 sql/lease.go:288  error releasing lease &{'4' '\x03' {{%!q(uint64=821537000) %!q(int64=63724793833) %!q(*time.Location=<nil>)}}}: lease-release: failed to send RPC: sending to all 3 replicas failed; last error: <nil> failed to connect to n3 at 127.0.0.1:46301: context canceled
W200511 11:32:28.720460 410692 kv/kvserver/raft_transport.go:637  [n1] while processing outgoing Raft queue to node 3: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200511 11:32:28.722143 410548 server/node.go:724  [n3,summaries] error recording status summaries: node unavailable; try another peer
W200511 11:32:28.723270 410192 gossip/gossip.go:1525  [n3] no incoming or outgoing connections
I200511 11:32:28.712072 420071 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n3: failed to connect to n3 at 127.0.0.1:46301: context canceled
W200511 11:32:28.742144 419966 kv/txn.go:607  failure aborting transaction: node unavailable; try another peer; abort caused by: failed to fetch protectedts metadata: failed to read metadata: protectedts-GetMetadata: context canceled
W200511 11:32:28.743422 410512 kv/kvserver/raft_transport.go:637  [n2] while processing outgoing Raft queue to node 1: rpc error: code = Canceled desc = grpc: the client connection is closing:
W200511 11:32:28.752246 410066 ts/db.go:194  [n2,ts-poll] error writing time series data: node unavailable; try another peer
E200511 11:32:28.778610 410337 jobs/registry.go:507  error while adopting jobs: failed querying for jobs: adopt-job: node unavailable; try another peer
I200511 11:32:28.798723 420088 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n2: failed to connect to n2 at 127.0.0.1:36989: context canceled
I200511 11:32:28.801799 420087 rpc/nodedialer/nodedialer.go:160  [ct-client] unable to connect to n1: failed to connect to n1 at 127.0.0.1:43493: context canceled
E200511 11:32:28.810429 410135 jobs/registry.go:507  error while adopting jobs: failed querying for jobs: adopt-job: node unavailable; try another peer
--- FAIL: TestProtectedTimestampsDuringImportInto (64.93s)
    sql_runner.go:200: error executing 'SELECT * FROM foo': pq: batch timestamp 1589196747.513939044,0 must be after replica GC threshold 1589196747.572016670,0

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestProtectedTimestampsDuringImportInto PKG=./pkg/ccl/importccl TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestProtectedTimestampsDuringImportInto.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/ccl/importccl/import_into_test.go
+++ b/pkg/ccl/importccl/import_into_test.go
@@ -54,7 +54,7 @@ func TestProtectedTimestampsDuringImportInto(t *testing.T) {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 	args := base.TestClusterArgs{}
-	tc := testcluster.StartTestCluster(t, 3, args)
+	tc := testcluster.StartTestCluster(t, 1, args)
 	defer tc.Stopper().Stop(ctx)
 
 	tc.WaitForNodeLiveness(t)

